lib/bin/run-server.js

```diff
@@ -32,8 +32,11 @@ const mail = mailer(merge(config.get('default.email'), { env }));
 const googler = require('../external/google');
 const google = googler(config.get('default.external.google'));
 
-// get a sentry and xlsform client, and a password module.
+// get a sentry and configure errors.
 const Sentry = require('../external/sentry').init(config.get('default.external.sentry'));
+Error.stackTrackLimit = 20;
```

Comment:
The default `stackTraceLimit` is 10, so I bumped it to 20. See https://nodejs.org/docs/latest-v14.x/api/errors.html#errors_error_stacktracelimit